internal/cli/capact/parameters.go

```diff
@@ -160,6 +165,38 @@ func (i *InputParameters) ResolveVersion() error {
 			i.Override.CapactValues.Global.ContainerRegistry.Tag = LocalDockerTag
 		}
 	}
+
+	// if not already set via flags, resolve base on our logic
+	if i.ActionCRDLocation == "" {
+		if err := i.resolveCRDLocationFromVersion(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// resolveCRDLocationFromVersion sets the CRD location.
+// If version was:
+// - local tag, use the relative local CRD path
+// - stable release (tag), use tag
+// - the latest release from main (tag-commit), use the commit sha
+func (i *InputParameters) resolveCRDLocationFromVersion() error {
+	if i.Version == LocalVersionTag {
+		i.ActionCRDLocation = LocalCRDPath
+		return nil
+	}
+
+	decoded, err := semver.NewVersion(i.Version)
+	if err != nil {
+		return err
```

Comment:
please wrap this error, e.g.:

```suggestion
		return errors.Wrap(err, "while parsing SemVer version")
```